docs/src/starknet/account.md

```diff
@@ -82,18 +90,10 @@ Salt will not be randomly generated if it's specified with `--salt`.
 ```shell
 $ sncast \
     account create \
+    --url http://127.0.0.1:5050 \
```

Comment:
I'd move `### ``account create`` With Salt Argument` to advanced use cases as well.